docs/source/overview/index.rst

```diff
@@ -22,4 +22,12 @@ Introduces the Traffic Control architecture, components, and their integration.
 	:maxdepth: 2
```

Comment:
This isn't exactly related to what you're trying to fix, but if you change this to `3` I think the properties of Parameters and Profiles would appear in the sidebar. Which could be nice.